apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsClaim.java

```diff
@@ -0,0 +1,106 @@
+package gov.cms.bfd.model.rda;
+
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Set;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.FieldNameConstants;
+
+@Entity
+@Getter
+@Setter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@FieldNameConstants
+@Table(name = "`McsClaims`", schema = "`pre_adj`")
```

Comment:
I think we're missing `idrTotBilledAmt`

Reply:
and `idrClaimMbi`

Reply:
Good catch.  My script didn't convert the decimal columns.  I have pushed a fix with the extra 4 fields.